fixtures/Booking.proto

```diff
@@ -73,8 +73,7 @@ message Booking {
   /** Has booking confirmation been sent? */
   required bool confirmation_sent = 4;
 
-  /** Has payment been received? */
-  optional bool payment_received = 5 [default = false, (com.pseudomuto.protokit.v1.extend_field) = true];
```

Comment:
Was this removed intentionally?

Reply:
🤔  No, not quite sure what happened there but I'll revert it.